hackclub.com.yaml

```
@@ -1263,6 +1263,10 @@ globalcovid:
   ttl: 600
   type: CNAME
   value: globalcovid.netlify.com.
+golden:
+  - ttl: 600
+    type: CNAME
+    value: https://kriaga259.github.io/golden/.
```

DNS records can't have protocols (`https`) or paths (`/golden/`)

```suggestion
    value: kriaga259.github.io.
```